Software-Construction/Factory-And-Abstract-Factory/Graphics/Point.java

```diff
@@ -0,0 +1,34 @@
+package Graphics;
+
+/**
+ * @author Niraj Gupta
+ * Class to represent the Cartesian point on the 2-D plane. 
+ */
+public class Point{
+	private double x, y;

```

Comment:
can we here use more meaningful variable names that x and y ?

Reply:
Cartesian points are represented by x and y. What else can be used in place of x and y?